### PR TITLE
Remove leftovers after removing old IDP RBACs

### PR DESCRIPTION
**Pull request by feichashao**
Old RBAC were removed.
https://github.com/openshift/managed-cluster-config/pull/894

However, there are still some RBACs in operators referring those groups.

For must-gather, the needed permission is already covered by backplane.
https://github.com/openshift/managed-cluster-config/blob/8bef2e7b796fc7fdd24ee693029c96302cfa6fc6/deploy/backplane/cee/20-cee-mustgather.Role.yml

For packet capture, the needed groups are already removed, so the roles and rolebindings are no longer valid. We will create a card for backplane to track the packet capture.

For prometheus rule,  it is to alert long time elevation using old idp. As those groups were removed and transited to backplane, it is not needed anymore.

### CODE REVIEW

**Comment by feichashao** [comment]
/assign @dustman9000

Hey @dustman9000 , could you help to review when you have time?
Thanks!

File: deploy/osd-pcap-collector/05-pcap-dedicated-admins-ClusterRole.yaml [hunk 1]
```diff
@@ -1,13 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
```
Comment (dustman9000) [inline deploy/osd-pcap-collector/05-pcap-dedicated-admins-ClusterRole.yaml:1]: This ClusterRole needs to remain to allow customers to run pcaps as dedicated admin.

File: deploy/osd-pcap-collector/06-pcap-dedicated-admins-ClusterRoleBind.yaml [hunk 2]
```diff
@@ -1,11 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
```
Comment (dustman9000) [inline deploy/osd-pcap-collector/06-pcap-dedicated-admins-ClusterRoleBind.yaml:1]: This ClusterRoleBinding needs to remain to allow customers to run pcaps as dedicated admin.

File: deploy/osd-pcap-collector/04-pcap-dedicated-admins-scc.yaml [hunk 3]
```diff
@@ -1,13 +0,0 @@
-kind: SecurityContextConstraints
```
Comment (dustman9000) [inline deploy/osd-pcap-collector/04-pcap-dedicated-admins-scc.yaml:1]: This SCC should not be deleted as this is needed for customers running as dedicated-admins to collect pcaps.

File: deploy/osd-pcap-collector/05-pcap-dedicated-admins-ClusterRole.yaml (hunk 1 above)
Comment (feichashao) [reply deploy/osd-pcap-collector/05-pcap-dedicated-admins-ClusterRole.yaml:1]: Thank you for pointing this out!  It shouldn't be removed. I don't know why I removed the whole directory..... I will review again when my brain cool down.

**Comment by feichashao** [comment]
@dustman9000  Revert the dedicated-admin files.

**Pull request merged**